_layouts/default.html

```diff
@@ -13,7 +13,9 @@
 <body class="d-flex flex-column vh-100" data-bs-theme="light">
 {% include nav.html %}
 <div class="container-fluid" data-bs-theme="light">
+    <div class="row">
```

Comment:
This change breaks the quickstart and blog posts layouts, as the gutters in the `card` are off which causes the `card-header` background to render incorrectly. Could this `div` not move to the `index.html` layout?

Alternately, adding `px-0` to the parent container and omitting this `row` entirely seems to solve the issue of excess x-padding on the hero.

Reply:
Thanks for double checking this. I have limited knowledge of working with bootstrap. Thanks for the fix!

Reply:
I've submitted a [PR](https://github.com/insectengine/kroxylicious.github.io/pull/1) on your branch to fix this.